authui/src/authflowv2/components/radio.css

```diff
@@ -0,0 +1,69 @@
+@layer components {
+  :root {
+    --radio__border-color: var(--color-neutral-500);
+    --radio__border-color--checked: var(--color-primary-theme-primary);
+    --radio__bg-color: var(--color-surface-primary);
+
+    --radio__text-color: var(--color-neutral-700);
+    --radio__font-family: var(--typography-label-large__font-family);
+    --radio__font-size: var(--typography-label-large__font-size);
+    --radio__line-height: var(--typography-label-large__line-height);
+    --radio__letter-spacing: var(--typography-label-large__letter-spacing);
+    --radio__font-weight: var(--typography-label-large__font-weight);
+
+    --radio__label-border-color: var(--color-neutral-100);
+  }
+  :root.dark {
+    --radio__border-color: var(--color-neutral-500);
+    --radio__border-color--checked: var(--color-primary-theme-primary);
+    --radio__text-color: var(--color-neutral-100);
+
+    --radio__label-border-color: var(--color-neutral-400);
+  }
+
+  .radio {
+    @apply w-5 h-5;
+    @apply rounded-full;
+    @apply border-2 border-solid;
+    border-color: var(--radio__border-color);
+  }
+
+  .radio:checked {
+    border-color: var(--radio__border-color--checked);
+    background-color: var(--radio__border-color--checked);
+    box-shadow: inset -0.005rem -0.005rem 0 0.15rem var(--radio__bg-color);
+  }
+
+  .radio:not(:checked):hover {
+    border-color: var(--radio__border-color);
+    background-color: var(--radio__border-color);
+    box-shadow: inset -0.005rem -0.005rem 0 0.15rem var(--radio__bg-color);
+  }
+
+  .radio-label {
+    @apply grid grid-cols-[auto,1.25rem] gap-y-4;
+    @apply px-2 py-4;
+    @apply w-full;
```

Comment:
You applied `radio-label` to a div and by default a div will already stretch to fill parent so this style is unnecessary